documentation/src/infrastructure/setup/requirements/frontend.md

```diff
@@ -1,23 +1,37 @@
 # Frontend one-time setup
 
+## Requirements
+
+- NodeJS v15
+    - [NodeJS](https://nodejs.org/en/)
+    - Or with [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md#installing-and-updating)
+        - `nvm install v15 && nvm alias default v15`
+- rustc nightly _after_ [edition 2021 was stabilized](https://github.com/rust-lang/rust/pull/88100#event-5229813170)
+    - [Install](https://www.rust-lang.org/tools/install)
+    - Update if necessary: `rustup default nightly && rustup update`
+- [cargo-make](https://github.com/sagiegurari/cargo-make)
+    - `cargo install cargo-make`
+- [cargo-watch](https://github.com/watchexec/cargo-watch)
+    - `cargo install cargo-watch`
+- [watchexec-cli](https://github.com/watchexec/watchexec)
```

Comment:
Doesn't really seem like watchexec is still used by anything on the frontend.

Reply:
Than please remove it